olp-cpp-sdk-core/include/olp/core/context/EnterBackgroundSubscriber.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2023 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+namespace olp {
+namespace context {
+
+/**
+ * @brief Used only for the iOS environment to be informed that the application
+ * is going to background
+ *
+ * @param enter Equals `true` to enter background mode and `false` to exit
+ * background mode.
+ *
+ * Initialize the `Context` class before you send network requests in
+ * the iOS environment.
```

Comment:
Add this one to the Context, since Context itself warns users only about Android requests